applications/FluidDynamicsApplication/custom_elements/two_fluid_navier_stokes.cpp

```diff
@@ -91,6 +89,14 @@ void TwoFluidNavierStokes<TElementData>::CalculateLocalSystem(
             Matrix shape_functions_enr_pos, shape_functions_enr_neg;
             GeometryType::ShapeFunctionsGradientsType shape_derivatives_pos, shape_derivatives_neg;
             GeometryType::ShapeFunctionsGradientsType shape_derivatives_enr_pos, shape_derivatives_enr_neg;
+            Matrix int_shape_function_neg;                                       // interface shape functions
+            Matrix int_shape_function_enr_neg, int_shape_function_enr_pos;       // interface enriched shape functions
+            GeometryType::ShapeFunctionsGradientsType int_shape_derivatives_neg; // interface shape functions derivatives
+            Vector int_gauss_pts_weights;                                // interface Gauss points weights
+            std::vector<Vector> int_normals_neg;                                 // interface normal vector based on the negative side
+            Vector gauss_pts_curvature;                                  // curvatures calculated on interface Gauss points
+
+            ModifiedShapeFunctions::Pointer p_modified_sh_func = ModifiedShapeFunctionsUtility(p_geom, data.Distance);
```

Comment:
I'd use a `UniquePointer` for this.

Reply:
Done.

Reply:
Changing it to unique pointer (and changing the input type for ComputeSplitting accordingly) results in: **error: use of deleted function**
pGetModifiedShapeFunctionsUtility returns a unique pointer, maybe this makes a conflict?